src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException(String.valueOf(number));
```

Comment:
```suggestion
            throw new IllegalArgumentException("Some message" + number);
```
always give some custom message to exception. it makes easier to handle errors